index.js

```diff
@@ -26,6 +26,7 @@ module.exports = defaultGet => {
 	function getSheet(key, id, get = defaultGet) {
 		return get(buildSheetUrl(key, id)).then(sheetData => {
 			const feed = sheetData.feed
+			if (feed.entry === undefined) feed.entry = [];
 			const rows = feed.entry.map(entry => {
```

Comment:
How about a bit of cleverness here?

```suggestion
			const rows = (feed.entry || []).map(entry => {
```